lnwallet/wallet.go

```diff
@@ -457,8 +457,36 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 		return
 	}
 
-	capacity := req.LocalFundingAmt + req.RemoteFundingAmt
-	localFundingAmt := req.LocalFundingAmt
+	var (
+		coins  []*wire.TxIn
+		change []*wire.TxOut
+		err    error
+	)
+
+	// If we're on the receiving end of a single funder channel then we
+	// don't need to perform any coin selection, and the remote contributes
+	// all funds.
+	capacity := req.RemoteFundingAmt
+	localFundingAmt := btcutil.Amount(0)
+
+	// Otherwise, attempt to obtain enough coins to meet the required
+	// funding amount.
+	if req.LocalFundingAmt != 0 {
```

Comment:
Add TODO to handle dual funder case?

Reply:
I think dual funder is a known TODO tbh :p